site/docs/guides/try-nessie.md

```diff
@@ -0,0 +1,48 @@
+# Getting Started with Nessie and Iceberg REST
+
+Nessie supports Apache Iceberg REST! Some of the features are
```

Comment:
```suggestion
"Nessie Catalog" is the new Nessie-backed implementation of the Apache Iceberg REST Catalog! Some of its features are:
```

Reply:
Why introduce a new "Nessie _Catalog_" to users?